### DISCUSSION TITLE
SageTV Android Miniclient no longer found on Google Play Store

### DISCUSSION THREAD

**Discussion by Swagger200** [OpenSageTV/Community#131] [Android client]
Has anyone else noticed this? Is this because the developer has abandoned the app? Sideloading looks painful, so I am hoping this appears again. I use the iminiclient on Samsung tablets and the Nvidia Shield Pro, and by and large it has worked well.

**Comment by TGsell1** [comment]
Agree the miniclient is crucial for my use.  Sent from my iPhoneOn Apr 28, 2024, at 2:13 PM, Swagger200 ***@***.***> wrote:﻿
Has anyone else noticed this? Is this because the developer has abandoned the app? Sideloading looks painful, so I am hoping this appears again. I use the iminiclient on Samsung tablets and the Nvidia Shield Pro, and by and large it has worked well.

**Comment by nyplayer** [comment]
Eventually this will stop working ... As Google keeps upgrading the Android Libraries.

**Comment by emveepee** [comment]
It's not really the libraries it would be compile options https://support.google.com/googleplay/android-developer/answer/11926878 I am surprised it worked this far. In any case sideloading is actually pretty easy and it's not like the app is getting updated frequently.

**Reply by nyplayer** [reply]
SageTVMiniClient users are going to have to make decisions on whether to continue with Sage as the FireTV's also will be moving to a new OS. Might have to go the route of MINIPC's on each TV.... or move to a DVR that is being supported.

**Reply by wayner9** [reply]
Or find someone willing to sell SageTV extenders, as long as you don't mind one box for SageTV and another box for other stuff.

**Comment by way-lo** [comment]
> Has anyone else noticed this? Is this because the developer has abandoned the app? Sideloading looks painful, so I am hoping this appears again. I use the iminiclient on Samsung tablets and the Nvidia Shield Pro, and by and large it has worked well.

Sideloading is extremely easy on tablets.  On the Shield Pro it takes some doing but it is not difficult either.  We'll see what happens to its availability on the appstore.

**Reply by LoisGNS** [reply]
Agree about sideloading being no problem on phones & tablets - a bit trickier on some other devices, but still doable. I have enough extenders that I don't use the miniclient often, but I have relied on it when I need to quickly check a recording's status while away from home, or other times when it's just a little more convenient to use my phone or tablet. I don't often use it for extensive watching these days, but would certainly be unhappy if I could no longer use it.

**Comment by jzhvymetal** [comment]
Sideloading is easy as long as you know how to use samba or windows filesharing

1.  Turn on developer mode by clicking about version. (I think it is 5 time but just keep clicking) 
2.  Download ES filemanager
3.  Add a network location where APK file is located
4.  open and run the APK file

The same can be done if you use appdownload and host it on website.

**Comment by TGsell1** [comment]
Agree.  But I’mNot sure I have the latest file for the client to sideload.  Does anyone have a copy to share.  If it’s not on the Google store any longer lots of folks will want a copy.  Thanks Sent from my iPhoneOn May 2, 2024, at 9:30 PM, jzhvymetal ***@***.***> wrote:﻿
Sideloading is easy as long as you know how to use samba or windows filesharing

Turn on developer mode by clicking about version. (I think it is 5 time but just keep clicking)
Download ES filemanager
Add a network location where APK file is located
open and run the APK file

The same can be done if you use appdownload and host it on website.

**Reply by jzhvymetal** [reply]
is this not the latest?

https://github.com/OpenSageTV/sagetv-miniclient/releases

**Comment by TGsell1** [comment]
Yes thanks I was a little confused about where to find it but thanks for clarifying.  I’m tucking this away in case everything disappears! lolSent from my iPhoneOn May 2, 2024, at 9:54 PM, jzhvymetal ***@***.***> wrote:﻿
is this not the latest?
https://github.com/OpenSageTV/sagetv-miniclient/releases

**Comment by Swagger200** [comment]
Thanks for the how-to, think I can deal with that.Sent from my iPhoneOn May 2, 2024, at 9:30 PM, jzhvymetal ***@***.***> wrote:﻿
Sideloading is easy as long as you know how to use samba or windows filesharing

Turn on developer mode by clicking about version. (I think it is 5 time but just keep clicking)
Download ES filemanager
Add a network location where APK file is located
open and run the APK file

The same can be done if you use appdownload and host it on website.

**Comment by Sagetvmaster** [comment]
Has anyone heard or seen Josh in the last year?  He was doing such a great job with the miniclient.  Did he surface at some point and say "I'm done with this, sorry", or has anyone contributed to his patreon effort and know what's up with his whereabouts?

**Comment by TGsell1** [comment]
I support his Patreon page with the hopes he will re-engage.  He does respond via that site.  Agree the mini client is an incredible pc of work.  Sent from my iPhoneOn Aug 3, 2024, at 6:34 AM, Sagetvmaster ***@***.***> wrote:﻿
Has anyone heard or seen Josh in the last year?  He was doing such a great job with the miniclient.  Did he surface at some point and say "I'm done with this, sorry", or has anyone contributed to his patreon effort and know what's up with his whereabouts?

**Comment by LoisGNS** [comment]
I have miniclient on my phone & tablets, which is what I mostly use when I run it. I've put it on my Fire stick and probably a couple of other devices, but haven't checked them recently. My phone uses fer 1.2.8, and I think each of my tablets uses something a little different. I have found that each device needs a different version. On the wrong version, I get a very garbled display once I connect to the server. So I keep a full set of apks around to sideload as needed.

**Comment by wayner9** [comment]
FYI - you should be able to install this by going to the Google Play Store web page on a PC and then installing this app on additional devices.  Whether it still works is another question.

**Comment by wayner9** [comment]
For me the SageTV app was still there in my installed apps.  I clicked on my profile in the top right and then on Libraries and Devices and every Android app that I have ever installed is listed there, including a bunch of Blackberry apps that are god knows how old.  Then I click on the app, in this case "SageTV MiniClient - Android Easy Apps Team Member" and then it goes into another screen where you can install the app on other devices.
